test/router.ts

```diff
@@ -0,0 +1,317 @@
+var assert = require("assert");
+var router = require("../lib/router");
+
+describe("router.Router", () => {
+  describe("constructor", () => {
+    it("parses well formatted configs", () => {
+      process.env.SCHOOL = "Hogwarts";
+      const conf = `
+routes:
```

Comment:
I thought the config was supposed to go in the launch yml under the `routes` key?